### ISSUE TITLE
New device does not handle blank shadow.

### ISSUE THREAD

**Issue by coderbyheart** [hello-nrfcloud/firmware#197]
```bash
[00:05:16.916,107] <err> app: Ignoring incoming configuration change due to decoding error: 10
```

[shadow.json](https://github.com/user-attachments/files/15925402/shadow.json)

**Comment by simensrostad** [comment]
This should be addressed in the latest version.

**Comment by coderbyheart** [comment]
Testing with https://github.com/hello-nrfcloud/firmware/releases/tag/v2.0.0-preview.12

**Comment by coderbyheart** [comment]
Fixed in [v2.0.0-preview.12](https://github.com/hello-nrfcloud/firmware/releases/tag/v2.0.0-preview.12)
